src/frontend/sharedTypes/navigation.ts

```diff
@@ -96,6 +96,7 @@ export type RootStackParamsList = {
   SaveTrack: undefined;
   Sync: undefined;
   Track: {trackId: string};
+  TrackEdit: {trackId: string} | undefined;
```

Comment:
when would `undefined` happen?

Reply:
I was following the example of ObservationEdit.
I was thinking it could be possible for someone to delete the track and maybe then quickly try to edit it before the deletion has gone through? It was just a guess. Mostly just following example of ObservationEdit

Reply:
i _think_ in the case of the observation flow, it's accounting for when you close the app and re-open it. i believe we maintain the navigation state while you're in that flow so that you can return to it immediately after re-opening the app. i don't think we're doing the same thing for the tracks flow at the moment although that's probably something we'd like to do in the future.

if we're not currently saving the nav state for the tracks-specific flow, I would lean towards removing the related code and re-introducing it later if we end up deciding on copying that approach.